src/ontology/mondo-edit.obo

```diff
@@ -104995,20 +104995,23 @@ is_a: MONDO:0005560 {source="DOID:4267", source="EFO:0007138", source="MESH:D000
 id: MONDO:0005641
 name: aleutian mink disease
 def: "A slow progressive disease of mink caused by the aleutian mink disease virus. It is characterized by poor reproduction, weight loss, autoimmunity, hypergammaglobulinemia, increased susceptibility to bacterial infections, and death from renal failure. The disease occurs in all color types, but mink which are homozygous recessive for the Aleutian gene for light coat color are particularly susceptible." [MESH:D000453]
-subset: otar {source="MONDO:OTAR"}
-synonym: "Aleutian disease" EXACT [DOID:2934]
-synonym: "Aleutian mink disease virus caused disease or disorder" EXACT [MONDO:patterns/specific_infectious_disease_by_agent]
-synonym: "Aleutian mink disease virus disease or disorder" EXACT []
-synonym: "Aleutian mink disease virus infectious disease" EXACT []
-xref: DOID:2934 {source="EFO:0007139", source="MONDO:equivalentTo"}
-xref: EFO:0007139 {source="MONDO:equivalentTo", source="MONDO:EFO"}
+comment: This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:19961696). In Mondo, we distinguish between human and non-human animal diseases, and we currently do not have a general term to represent a species-agnostic version of the disease. This record is for the non-human version of “aleutian mink disease”.
+synonym: "Aleutian disease" RELATED [DOID:2934]
+synonym: "Aleutian mink disease virus caused disease or disorder" RELATED [MONDO:patterns/specific_infectious_disease_by_agent]
+synonym: "Aleutian mink disease virus disease or disorder" RELATED [] {source="https://orcid.org/0000-0002-5002-8648"}
+synonym: "Aleutian mink disease virus infectious disease" RELATED [] {source="https://orcid.org/0000-0002-5002-8648"}
```

Comment:
yep! you should use dbx-ref and not "source" for synonyms

Reply:
done